superdesk/vocabularies_async/service.py

```diff
@@ -58,14 +58,14 @@ class VocabulariesService(AsyncResourceService[VocabulariesResourceModel]):
 
     async def _validate_items(self, update: VocabulariesResourceModel) -> None:
         # if we have qcode and not unique_field set, we want it to be qcode
-        if update.schema_.get("qcode") and update.unique_field is None:
+        if update.schema.get("qcode") and update.unique_field is None:
             update.unique_field = "qcode"
 
         unique_field = update.unique_field
         vocabs = {}
-        if update.schema_ and update.items:
+        if update.schema and update.items:
```

Comment:
I did use `schema_` because `schema` is already used in `ResourceModel`. With your change, I have that:
```
/home/goffi/dev/sourcefabric/venv/async_3.10/lib/python3.10/site-packages/pydantic/_internal/_fields.py:172: UserWarning: Field name "schema" in "VocabulariesResourceModel" shadows an attribute in parent "ResourceModel"
```

Reply:
I'm reverting to `schema_` and using:
```python
schema_: Annotated[dict[str, dict], Field(alias="schema")]
```